docs/topics/using/edgectl/service-preview-install.md

```diff
@@ -2,42 +2,231 @@
 
 Service Preview is installed as an addon to the Ambassador Edge Stack.
 
-There are two main mechanisms for installing Service Preview
-
-- [`edgectl install`](#install-with-edgectl) will boot strap your cluster with the Ambassador Edge Stack and Service Preview and only works when doing a fresh install of Ambassador.
-
-- [`YAML Install`](#install-with-yaml) will work with an existing installation of Ambassador Edge Stack.
-
 ## Prerequisites
 
 - [`kubectl`](https://kubernetes.io/docs/tasks/tools/install-kubectl/) access to a Kubernetes cluster
-- The Ambassador Edge Stack [Installed](../../../../tutorials/getting-started/)
 - [`edgectl`](../edge-control#installing-edge-control) client
 
-## Install with Edgectl
+## Install
+
+There are three method for installing Service Preview.
+
+### [<img class="Ambassador's OpenSource Blackbird" src="../../../images/features-page-bird.svg"/> Install with Edgectl](#install-with-edgectl)
+
+If you are installing Service Preview and Ambassador Edge Stack for the first time, `edgectl` will bootstrap your cluster with the Ambassador Edge Stack and Service Preview automatically.
+
+### [<img class="k8s-logo" src="../../../images/kubernetes.png"/> Install with YAML](install with yaml)
+
+The YAML installation method will walk you through a step-by-step deployment of all the resources necessary for installing Service Preview alongside the Ambassador Edge Stack. The YAML installation method is the most common approach to install Ambassador Edge Stack, especially in production environments, with our default, customizable manifest.
```

Comment:
> The YAML installation method is the most common approach

Although it is true, I'm afraid mentioning it will lead people to choose this option instead of _our_ preferred edgectl install option.

Reply:
So I have this because it is wording in the AES install doc 

https://www.getambassador.io/docs/latest/topics/install/#img-classos-logo-srcimageskubernetespng-install-via-kubernetes-yaml

> Kubernetes via YAML is the most common approach to install Ambassador Edge Stack, especially in production environments, with our default, customizable manifest. So if you want complete configuration control over specific parameters of your installation, use the manual YAML installation method.